test/fixtures/krb5kdc-fixture/build.gradle

```diff
@@ -32,5 +32,26 @@ tasks.named("postProcessFixture").configure {
   }
 }
 
+
```

Comment:
We'll remove other usages of project.ext... in upcoming pull requests and remove this to ensure we use gradle idiomatic api for resolving these fixture artifacts

Reply:
:+1: